### PR TITLE
connect post creation to redux

### PR DESCRIPTION
**Pull request by Amym123**

### CODE REVIEW

**Comment by Amym123** [comment]
@jameskanCa is connecting like this what you were talking about?

File: client/src/redux/actions/inquiryActionsCreators.js [hunk 1]
```diff
@@ -44,31 +44,32 @@ export const GetInquiries = (userId) => {
...
+      //         suggestionTitle: 'TEST THIS!',
+      //         suggestionDescription: 'It will not look great on you',
+      //         imageIndex: 2,
+      //       },
+      //     ],
+      //   };
+      return dispatch(AddPost(userId, inquiry, postId));
+      // }, 2000);
     } catch (error) {
+      console.log('here');
```
Comment (jameskanCa) [inline client/src/redux/actions/inquiryActionsCreators.js:72]: don't forget to remove

File: client/src/components/PostCreation/InquiryPostCreationComponent.js [hunk 2]
```diff
@@ -65,4 +73,13 @@ export default function InquiryPostCreationComponent() {
...
+};
+
+const mapStateToProps = (state) => {
+  console.log(state);
+  return {
+    userId: state.userReducer.profile.userId,
+  };
+};
+
+export default connect(mapStateToProps, {})(InquiryPostCreationComponent);
```
Comment (jameskanCa) [inline client/src/components/PostCreation/InquiryPostCreationComponent.js:85]: You can instead of {} just give it a null but that's up to you

Comment (jameskanCa) [reply client/src/components/PostCreation/InquiryPostCreationComponent.js:85]: or just have nothing :)

File: client/src/redux/actions/postAction.js [hunk 3]
```diff
@@ -6,11 +6,12 @@ export const LoadingPosts = () => {
   };
 };
 
-export const AddPost = (userId, post) => {
+export const AddPost = (userId, post, postId) => {
```
Comment (jameskanCa) [inline client/src/redux/actions/postAction.js:9]: Can the postId be inside the new post? as a field? 

In addition what about the imageIndex, can you mock either values 1 or 2 at random? we only have 2 photos currently in front end

File: client/src/components/PostCreation/InquiryPostCreationComponent.js (hunk 2 above)
Comment (Amym123) [reply client/src/components/PostCreation/InquiryPostCreationComponent.js]: deleted {}

File: client/src/redux/actions/postAction.js (hunk 3 above)
Comment (Amym123) [reply client/src/redux/actions/postAction.js]: added imageIndex and postId into post object

**Pull request merged**